src/routes/search/components/SearchModal.js

```diff
@@ -19,20 +20,23 @@ const Wrapper = styled.View`
 
 type CategoryListItemType = {| model: CategoryModel, subCategories: Array<CategoryModel> |}
 
-type PropsType = {|
+export type PropsType = {|
   categories: CategoriesMapModel | null,
-  navigateToCategory: (cityCode: string, language: string, path: string) => void,
-  theme: ThemeType,
-  language: string,
-  cityCode: string,
-  closeModal: () => void
+  navigateToCategory: (cityCode: string, language: string, path: string) => StoreActionType,
```

Comment:
There should be a more specific type for this, shouldn't it?

Reply:
I tried to use the specific type but it does not work with navigateToCategory right now